boards/arm/nrf52833_pca10114/nrf52833_pca10114.dts

```diff
@@ -22,27 +22,27 @@
 	leds {
 		compatible = "gpio-leds";
 		led0_green: led_0 {
-			gpios = <&gpio0 15 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 15 GPIO_INT_LOW_0>;
 			label = "Green LED 0";
 		};
 		led1_red: led_1 {
-			gpios = <&gpio0 20 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 20 GPIO_INT_LOW_0>;
 			label = "Red LED 1";
 		};
 		led1_green: led_2 {
-			gpios = <&gpio0 28 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 28 GPIO_INT_LOW_0>;
 			label = "Green LED 1";
 		};
 		led1_blue: led_3 {
-			gpios = <&gpio0 29 GPIO_INT_ACTIVE_LOW>;
+			gpios = <&gpio0 29 GPIO_INT_LOW_0>;
 			label = "Blue LED 1";
 		};
 	};
 
 	buttons {
 		compatible = "gpio-keys";
 		button0: button_0 {
-			gpios = <&gpio0 17 GPIO_PUD_PULL_UP>;
+			gpios = <&gpio0 17 GPIO_PULL_UP>;
```

Comment:
In the button configuration we could add `GPIO_ACTIVE_LOW` for consistency with other boards. This should be done for both mentioned boards.

Reply:
> In the button configurations we could use `(GPIO_PULL_UP | GPIO_ACTIVE_LOW)` for consistency with other boards.

Not only we could, we need to. Otherwise this definition will be incorrect.

Reply:
done.